tools/command_ignorelist

```diff
@@ -0,0 +1,521 @@
+journalctl.xml /refsect1[title="Output Options"]/variablelist/varlistentry[term="-a"]
```

Comment:
this could use a comment at the top describing the format - the other ignore lists are obvious, but this one isn't quite that obvious